vars/alvariumCreate.groovy

```diff
@@ -0,0 +1,54 @@
+@GrabResolver(name='jitpack.io', root='https://jitpack.io/')
+@Grab("com.google.errorprone:error_prone_annotations:2.20.0") // fixes alvarium import error
+@Grab(group='com.github.project-alvarium', module='alvarium-sdk-java', version='d18f5aeadd') 
+@Grab("org.apache.logging.log4j:log4j-core:2.15.0")
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import com.alvarium.DefaultSdk;
+import com.alvarium.Sdk;
+import com.alvarium.SdkInfo;
+
+import com.alvarium.annotators.Annotator;
+import com.alvarium.utils.PropertyBag;
+
+def call(List<String> annotatorKinds, String artifactPath=null) {
+
+    if (annotatorKinds.contains('checksum')) {
```

Comment:
Is it better to have these nested ifs &&ed instead?

Reply:
Resolved in 025bcd8d4638c77c395474037431b8258402bd96.

Reply:
I think I have the access and I can do the edits from my side but wanted to ensure if it has a certain purpose also in alvariumTransit and alvariumMutate.

Reply:
aha I missed that :D

Reply:
Resolved in d879b6ed10db562b6478f848bfdb2f23bb2bf8e4